### PR TITLE
man: add pt_BR translation

### PR DESCRIPTION
**Pull request by xRuiAlves**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.pt_BR/common/man.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# man
+
+> Utilitário para aceder a páginas do manual.
```
Comment (schneiderl) [inline pages.pt_BR/common/man.md]:
```suggestion
> Utilitário para exibir páginas do manual.
```

File: pages.pt_BR/common/man.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Visualizar a página do manual de um comando:
+
+`man {{comando}}`
+
+- Visualizar a página da secção 7 do manual de um comando:
```
Comment (schneiderl) [inline pages.pt_BR/common/man.md]:
```suggestion
- Visualizar a página da seção 7 do manual de um comando:
```

File: pages.pt_BR/common/man.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# man
+
+> Utilitário para aceder a páginas do manual.
+
+- Visualizar a página do manual de um comando:
```
Comment (andrik) [inline pages.pt_BR/common/man.md]:
```suggestion
- Visualizar o manual de um comando:
```

File: pages.pt_BR/common/man.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Visualizar a página da secção 7 do manual de um comando:
+
+`man {{comando}}.{{7}}`
+
+- Visualizar o caminho procurado pelas páginas do manual:
+
+`man --path`
+
+- Visualizar a localização de uma página do manual de um comando:
```
Comment (andrik) [inline pages.pt_BR/common/man.md]:
```suggestion
- Visualizar o caminho do manual de um comando:
```

File: pages.pt_BR/common/man.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`man -w {{comando}}`
+
+- Procurar por páginas do manual contendo uma string de pesquisa:
```
Comment (andrik) [inline pages.pt_BR/common/man.md]:
```suggestion
- Procurar por manuais contendo um termo de pesquisa
```

File: pages.pt_BR/common/man.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+`man -k "{{string de pesquisa}}"`
```
Comment (andrik) [inline pages.pt_BR/common/man.md]:
```suggestion
`man -k "{{termo_de_pesquisa}}"`
```

File: pages.pt_BR/common/man.md (hunk 2 above)
Comment (xRuiAlves) [reply pages.pt_BR/common/man.md]: @schneiderl Ah, em grafia pt_PT escrevemos "secção" :P

**Comment by xRuiAlves** [comment]
@andrik Implemented requested changes :+1:

File: pages.pt_BR/common/man.md (hunk 6 above)
Comment (schneiderl) [reply pages.pt_BR/common/man.md]: Eu estava pensando sobre isso: será que devemos usar snake case nesse caso? Isso não faria o usuário pensar que não se podem ter espaços na string?

Comment (xRuiAlves) [reply pages.pt_BR/common/man.md]: @schneiderl de acordo!

**Comment by sbrl** [comment]
Next time, if you're translating lots of pages, I'd recommend making 1 big PR with 1 commit per page :-)

**Pull request merged**